motr/client.h

```diff
@@ -933,33 +935,32 @@ struct m0_pi_hdr {
 
 struct m0_md5_pi {
 
-#define PI_VALUE_LENGTH 16
-
 	/* header for protection info */
 	struct m0_pi_hdr hdr;
+#ifndef __KERNEL__
 	/* protection value computed for the current data*/
-	unsigned char pi_value[PI_VALUE_LENGTH];
+	unsigned char pi_value[MD5_DIGEST_LENGTH];
 	/* structure should be 32 byte aligned */
-	char pad[M0_CALC_PAD(sizeof(struct m0_pi_hdr)+ PI_VALUE_LENGTH, 32)];
+	char pad[M0_CALC_PAD(sizeof(struct m0_pi_hdr)+ MD5_DIGEST_LENGTH, 32)];
+#endif
 };
 
 struct m0_md5_inc_digest_pi {
 
-#define PI_DIGEST_LENGTH 92
-#define PI_VALUE_LENGTH 16
-
 	/* header for protection info */
 	struct m0_pi_hdr hdr;
+#ifndef __KERNEL__
 	/*digest of previous data unit, required for checksum computation */
-	unsigned char prev_digest[PI_DIGEST_LENGTH];
+	unsigned char prev_digest[sizeof(MD5_CTX)];
```

Comment:
We can change name to prev_digest to prev_context to avoid confusion with
unsigned char pi_value[MD5_DIGEST_LENGTH]

Reply:
Done.